src/modules/role/role.dto.ts

```diff
@@ -121,6 +152,9 @@ export class RoleDefinitionDTO implements IRoleDefinition {
   @IsDefined()
   issuer: IssuerDTO;
 
+  @IsDefined()
```

Comment:
Do you need `ValidatedNested` here too? My current understanding is that it is necessary, but I could be wrong. https://github.com/typestack/class-validator#validating-nested-objects

If it is needed, I think we should add it to the other properties of this class as well
```suggestion
  @IsDefined()
  @ValidatedNested()
  @Type(() => RevokerDTO)
```